helm/redis/templates/deployment.yaml

```diff
@@ -22,6 +22,9 @@ spec:
         - name: config-volume
           configMap:
             name: {{ include "redis.fullname" . }}
+        - name: devfs
```

Comment:
Only configure this when dsa enabled.

Reply:
I'll remove these, and move resources to deployment.yaml